<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>318</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>854</width>
    <height>318</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>854</width>
    <height>318</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kardano Grid [CE]</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
    <property name="maximumBlockCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="codingButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Coding</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="decodingButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Decoding</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beginButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copyButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy to clipboard</string>
    </property>
   </widget>
   <widget class="QLabel" name="howMuch">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0 / 16</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kardanoLine">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to Kardano Grid pattern TXT (ASCII) file:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beginButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>begin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputText</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshHowMuch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codingButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decodingButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAbout()</slot>
  <slot>quit()</slot>
  <slot>begin()</slot>
  <slot>clear()</slot>
  <slot>copy()</slot>
  <slot>refreshHowMuch()</slot>
  <slot>changed()</slot>
 </slots>
</ui>
